src/app/commands/scoap-squad/ScoapSquad.ts

```diff
@@ -0,0 +1,90 @@
+import {
+	// ApplicationCommandPermissionType,
+	CommandContext,
+	CommandOptionType,
+	SlashCommand,
+	SlashCreator,
+} from 'slash-create';
+// import roleIDs from '../../service/constants/roleIDs';
+import ServiceUtils from '../../utils/ServiceUtils';
+import CreateNewScoapPoll from '../../service/scoap-squad/CreateNewScoapPoll';
+import ValidationError from '../../errors/ValidationError';
+
+module.exports = class ScoapSquad extends SlashCommand {
+	constructor(creator: SlashCreator) {
+		super(creator, {
+			name: 'scoap-squad',
+			description: 'Create or delete a SCOAP Squad request',
+			guildIDs: process.env.DISCORD_SERVER_ID,
+			options: [
+				{
+					name: 'assemble',
+					type: CommandOptionType.SUB_COMMAND,
+					description: 'Create a SCOAP Squad request',
+					options: [],
+				},
+				
+			],
+			throttling: {
+				usages: 2,
+				duration: 1,
+			},
+			// defaultPermission: false,
+			// permissions: {
+			// 	[process.env.DISCORD_SERVER_ID]: [
+			// 		{
+			// 			type: ApplicationCommandPermissionType.ROLE,
```

Comment:
At minimum we would want level 1 members to have access to this command. I don't see guest pass users wants to execute a scoap squad. So that should should have access are

level 1
level 2
level 3
level 4
Admin
Genesis Squad

As more roles are added and governance changes, we'll have to remember to update these accordingly

Reply:
Updated accordingly

Reply:
Updated accordingly